java/src/main/java/com/logicalclocks/hsfs/FeatureGroup.java

```diff
@@ -115,80 +115,175 @@ public FeatureGroup(FeatureStore featureStore, int id) {
     this.id = id;
   }
 
+  /**
+   * Read the feature group into a dataframe.
+   * Reads the feature group by from the offline storage as Spark DataFrame on Hopsworks and Databricks.
```

Comment:
I don't see why we need to mention `Databricks` here. Seems like this was done on all the methods. Please remove it.